libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -854,6 +854,35 @@ def DetachDisk(self, disk: 'GoogleComputeDisk') -> None:
     response = request.execute()
     self.BlockOperation(response, zone=self.zone)
 
+  def Delete(self, delete_disks: bool = False) -> None:
+    """Delete an Instance.
+
+    Args:
+      delete_disks (bool): whether to also force delete all attached disks
+        (ignores the 'readonly' bit).
+    """
+    disks_to_delete = []
+    if delete_disks:
+      disks_to_delete = [
+          disk['source'].split('/')[-1] for disk in self.GetValue('disks')]
+
+    gce_instance_client = self.GceApi().instances()
+    logger.info(
+        self.FormatLogMessage('Deleted Instance: {0:s}'.format(self.name)))
```

Comment:
maybe Deleting? since the deletion may fail and this will leave a confusing log line.

Reply:
done